src/OrchardCore/OrchardCore.Abstractions/Shell/Builders/Extensions/ServiceProviderExtensions.cs

```diff
@@ -21,18 +21,4 @@ public static TResult CreateInstance<TResult>(this IServiceProvider provider, Ty
     {
         return (TResult)ActivatorUtilities.CreateInstance(provider, type);
     }
-
```

Comment:
Are we sure that the new net10 method works the same way?

Reply:
Good point, it seems to be very similar (essentially the same, checks one extra input for null): https://github.com/dotnet/runtime/blame/main/src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ServiceProviderKeyedServiceExtensions.cs#L41-L52 (runtime).